### PR TITLE
Add missing documentation for --help and --version, and a short "Gett…

### PR DESCRIPTION
**Pull request by davidplowman**
…ing Help" section

### CODE REVIEW

File: documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+=== Getting Help
+
+For further help with `libcamera` and the `libcamera-apps`, the first port of call will usually be the https://www.raspberrypi.org/forums/viewforum.php?f=43[Rasperry Pi Camera Forum]. Before posting, it's helpful to:
+
+* Ensure your https://www.raspberrypi.org/documentation/computers/os.html#using-apt[software is up to date].
```
Comment (lurch) [inline documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc:5]: Can you make this a relative-link please?

Comment (davidplowman) [reply documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc:5]: Sure... what would a relative link look like?  :)

Comment (lurch) [reply documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc:5]: `xref:os.adoc#using-apt[software is up to date]`
(which I just stole from https://github.com/raspberrypi/documentation/blob/develop/documentation/asciidoc/computers/using_linux/using-the-terminal.adoc )

Comment (davidplowman) [reply documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc]: I think that's done now. I had to use `xref:../computers/os.adoc#using-apt[blah blah]` though!

File: documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+=== Getting Help
+
+For further help with `libcamera` and the `libcamera-apps`, the first port of call will usually be the https://www.raspberrypi.org/forums/viewforum.php?f=43[Rasperry Pi Camera Forum]. Before posting, it's helpful to:
+
+* Ensure your xref:../computers/os.adoc#using-apt[software is up to date].
+
+* If you are using _Buster_  or an earlier version of Raspberry Pi OS where you have had to build `libcamera` and `libcamera-apps` for yourself, or indeed if you are using a later version but have built them for yourself anyway, then you may need to update those _git_ repositories and repeat the build process.
+
+* Make a note of your operating system version (`uname -a`).
```
Comment (lurch) [inline documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc:9]: Does this contain enough info to be able to tell if the user is using Buster or Bullseye, or is that extra info that the user should provide too?

Comment (davidplowman) [reply documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc:9]: I think `uname -a` gives us that

**Review by lurch — APPROVED** [review]

**Comment by lurch** [comment]
Merged into #2162

**Pull request merged**